src/main/java/com/avispl/symphony/dal/infrastructure/management/polycom/lens/common/PolyLensAggregatedMetric.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2023 AVI-SPL, Inc. All Rights Reserved.
+ */
+
+package com.avispl.symphony.dal.infrastructure.management.polycom.lens.common;
+
+/**
+ * PolyLensAggregatedDeviceMetric contains properties for monitoring aggregated devices
+ *
+ * @author Harry / Symphony Dev Team<br>
+ * Created on 4/23/2023
+ * @since 1.0.0
+ */
+public enum PolyLensAggregatedMetric {
+	SUPPORTS_SETTINGS(PolyLensConstant.SUPPORTS_SETTINGS),
```

Comment:
why we have to pass the constant from another class to enum?

Reply:
because constant value is used many times, so i move it to constant class

Reply:
> why we have to pass the constant from another class to enum?

because constant value is used many times, so i move it to constant class